java/hsfs/src/main/java/com/logicalclocks/hsfs/metadata/HopsworksExternalClient.java

```diff
@@ -119,11 +119,12 @@ public HopsworksExternalClient(CloseableHttpClient httpClient, HttpHost httpHost
   }
 
   HopsworksExternalClient(String host, int port, Region region,
-                          SecretStore secretStore, boolean hostnameVerification,
+                          SecretStore secretStore, boolean hostnameVerification, String keyStorePath,
```

Comment:
It's not clear to me why we need the keystore here. At best, the http client only needs the truststore (only if Hopsworks is using a self signed certificate). It doesn't need the keystore.

Also in the `flink/HopsworksConnection.java` we are overwriting this path anyway with the certificate path from the Flink configuration. Same happens in the Spark client.

I really don't think we need to pass the keyStorePath here.

Reply:
agreed. I was thinking if we should ever let users to pass certificate path from here. I will remove it now